code/game/objects/items/shrapnel.dm

```diff
@@ -66,3 +66,18 @@
 
 /obj/item/projectile/bullet/pellet/stingball/on_ricochet(atom/A)
 	hit_stunned_targets = TRUE // ducking will save you from the first wave, but not the rebounds
+
+/obj/item/projectile/bullet/pellet/capmine
+	name = "\improper AP shrapnel shard"
```

Comment:
Wouldn't this be better of as a subtype of shrapnel? otherwise the name/path is a bit misleading

Reply:
that's where the AP part comes from